sections/kubectl_version.zsh

```diff
@@ -0,0 +1,36 @@
+#
+#  Kubernetes (kubectl)
+#
+# Kubernetes is an open-source system for deployment, scaling,
+# and management of containerized applications.
+# Link: https://kubernetes.io/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_KUBECTL_VERSION_SHOW="${SPACESHIP_KUBECTL_VERSION_SHOW=true}"
```

Comment:
Since this is the slow section, shall we disable this one by default? Or maybe disable parent `kubectl` section and enable by default the two child ones?